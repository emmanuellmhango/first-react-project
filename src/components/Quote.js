import React from 'react';

const Quote = function () {
  return (
    <div className="spacing quote">
      <p>
        Mathematics is not about numbers, equations, computations, or algorithms;
        it is about understanding. - William Paul Thurston
      </p>
    </div>
  );
};

export default Quote;
