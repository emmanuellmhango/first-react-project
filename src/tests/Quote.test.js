import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

describe('Snapshot testing', () => {
  test('Testing Quote component', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
