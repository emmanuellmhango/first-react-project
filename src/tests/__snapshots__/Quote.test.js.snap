// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing Testing Quote component 1`] = `
<div
  className="spacing quote"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms; it is about understanding. - William Paul Thurston
  </p>
</div>
`;
